tests/observability/metrics/test_vms_metrics.py

```
@@ -509,3 +511,27 @@ def test_kubevirt_vmi_phase_transition_from_deletion_seconds_sum_windows(
             metric_name=KUBEVIRT_VMI_PHASE_TRANSITION_TIME_FROM_DELETION_SECONDS_SUM_SUCCEEDED,
             initial_value=initial_metric_value,
         )
+
+
+class TestVirtOperatorReadyStatus:
+    @pytest.mark.polarion("CNV-12378")
+    def test_kubevirt_virt_operator_ready_status(
+        self,
+        prometheus,
+        disabled_virt_operator,
```

Ruff (0.14.3)

Unused method argument: `disabled_virt_operator`

(ARG002)